src/python/diurnal/scripts/compositeDiurnalStatisticsWrapped.py

```diff
@@ -77,7 +77,7 @@ def compute(params):
                     nlats = dimensions[1]
                     nlons = dimensions[2]
                     deltaH = 24. / N
-                    dayspermo = tvarb.shape[0] / N
+                    dayspermo = tvarb.shape[0] // N
```

Comment:
Just to be safe, you might want to have `from __future__ import division`. This will use the Python 3 way of dividing, if that's what you wanted. I'm not too sure if that's what you were aiming for however.

Reply:
you're right @zshaheen updating now, I'm surpised py2 passed.